src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -102,17 +99,18 @@ class BlockingProcess(
 
   private val process = startProcess()
 
+  // This shutdown hook is required because the stdio actors will be blocked on a read. The only
+  // reliable means of unblocking them is to destroy the thing they're listening to.
+  context.actorOf(ShutdownHook.props(() => process.destroy()), "process-destroyer")
```

Comment:
Thanks Viktor. Can I get a PR from you, or at least raise another issue? I'm under the gun at the mo and don't have time for further experiments on this right now. Thanks